operations/tasks/upgrade-docker.yml

```diff
@@ -0,0 +1,49 @@
+---
+- name: Mark node as unschedulable
+  command: >
+    oadm manage-node {{ inventory_hostname }} --schedulable=false
+
+- name: Wait until no more pods are running in node
+  register: active_pods_result
+  shell: >
+    oadm manage-node {{ inventory_hostname }} --list-pods |
+    grep -v 'STATUS' |
+    grep -v 'Completed' |
+    grep -v 'Error' |
+    wc -l
+  until: '0' in active_pods_result.stdout_lines
+  changed_when: false # read-only command
+  environment: "{{ osbs_environment }}"
+  retries: "{{ osbs_wait_active_pods_retries }}"
+  delay: "{{ osbs_wait_active_pods_delay }}"
+
+- name: Update docker package
+  yum:
+    name: "{{ osbs_docker_version }}"
+    update_cache: true
+    state: present
+
+- name: Restart docker and atomic-* services
+  systemd:
+    state: restarted
+    name: "{{ item }}"
+  with_items:
+  - docker
+  - atomic-openshift-master-api
```

Comment:
master services don't need to be restarted - generally nodes don't have those installed

Reply:
Yeah, let's remove these.